### DISCUSSION TITLE
site bugs

### DISCUSSION THREAD

**Discussion by kjjd84** [elrumo/macOS_Big_Sur_icons_replacements#1515] [General]
when doing a search if "popular" is selected no results come up

when scrolling through icons the infinite scroll just wont load more results a majority of the time

when viewing someones profile e.g https://macosicons.com/#/u/epicdude2121

it says they have like 200 icons but theres only like 20 visible on their profile and no more load even when scrolling or hard reloading the page

also when scrolling through search results the same icons often show up multiple times

**Comment by efcjimbo** [comment]
Has the site been abandoned? Is this site an experimental web app? Why are the bugs so bad for website that hosts pngs.

**Comment by exp-lab-usa** [comment]
All of these issues are still present as of May 2025. Looking more and more like it got abandoned.

**Comment by kjjd84** [comment]
i switched to https://github.com/rikumi/iconsur

takes a bit of setup but does the job

**Reply by exp-lab-usa** [reply]
Interesting power user & developer solution for semi-automating the icon generation but it’s not a repository of custom icons. The default iOS search method worked for about 15% of my apps. Then I had to manually add my inputs for others (where I’d still need to reference this repository). I’ll have to play around with scripting it to automatically pull from my local icon repository which could be very helpful. Thanks for sharing!

As far as alternatives go, I’ve also found the app, **Replacicon** to be useful. It finds custom icon options for many apps, but best of all, it keeps the changes through updates. Well, except for my Setapp apps but I’m sure that’s just me fudging the permissions/settings lol.

**Comment by elrumo** [comment]
All those bugs have now been fixed
